docs/source/cuda.rst

```diff
@@ -110,7 +110,12 @@ Memory management
      reset_peak_memory_stats
      caching_allocator_alloc
      caching_allocator_delete
+     set_memory_tracker
+     disable_memory_tracker
```

Comment:
in follow PRs, creating a context manager for it.

Reply:
Sure. Thank you.